sources/HUBLiveServiceImplementation.m

```diff
@@ -128,24 +113,17 @@ - (void)handleBytesAvailableForStream:(NSInputStream *)stream
     }
     
     NSData * const data = [mutableData copy];
-    
-    if (self.viewController != nil && self.contentOperation != nil) {
-        self.contentOperation.JSONData = data;
+
+    HUBLiveContentOperation *existingContentOperation = self.contentOperation;
+    if (existingContentOperation != nil) {
+        existingContentOperation.JSONData = data;
         return;
     }
-    
-    NSURL * const viewURI = [NSURL URLWithString:@"hubframework:live"];
-    
+
     HUBLiveContentOperation * const contentOperation = [[HUBLiveContentOperation alloc] initWithJSONData:data];
     self.contentOperation = contentOperation;
-    
-    HUBViewController * const viewController = [self.viewControllerFactory createViewControllerForViewURI:viewURI
-                                                                                        contentOperations:@[contentOperation]
-                                                                                        featureIdentifier:@"live"
-                                                                                             featureTitle:@"Hub Framework Live"];
-    
-    self.viewController = viewController;
-    [self.delegate liveService:self didCreateViewController:viewController];
+
+    [self.delegate liveService:self didCreateContentOperation:contentOperation];
```

Comment:
No, as long as we hold on to the content operation (which the view controller will do) it will be reused by the live service and the delegate method is not called.